crates/test-utils/src/util.rs

```diff
@@ -1023,7 +1023,10 @@ fn test_redactions() -> snapbox::Redactions {
                 "[ESTIMATED_AMOUNT_REQUIRED]",
                 r"Estimated amount required:\s*(\d+(\.\d+)?)\s*[A-Z]{3}",
             ),
-            ("[REVIVE_VERSION]", r"Resolc v\d+\.\d+\.\d+(-[a-zA-Z0-9\.]+)?, Solc v\d+.\d+.\d+"),
+            (
+                "[REVIVE_VERSION]",
+                r"Resolc (v)?\d+\.\d+\.\d+(-[a-zA-Z0-9\.]+)?, Solc (v)?\d+.\d+.\d+",
```

Comment:
why do we need (v)?
if we want to be in sync with solc, we should have: `Resolc( version)`?

Reply:
(v)? is needed to cover for `compiler::spawn` trace. 
`(version )?` is not used in the foundry-compilers at least.